CRM/Contact/BAO/Query.php

```diff
@@ -3558,14 +3558,13 @@ public function street_address(&$values) {
     $n = trim($value);
 
     if ($n) {
-      $value = strtolower($n);
```

Comment:
In fact this is a null change - we can see from tests / code step through strtolower is already done on this string

Reply:
@seamuslee001 note in some places we see

```
      $strtolower = function_exists('mb_strtolower') ? 'mb_strtolower' : 'strtolower';
      $strtolower($n);
```

Likely those places are more reliable